truss-chains/tests/chains_test.py

```diff
@@ -25,12 +25,45 @@ def test_chain():
 
         url = service.run_url.replace("host.docker.internal", "localhost")
 
+        # Call without providing values for default arguments.
+        response = requests.post(url, json={"length": 30, "num_partitions": 3})
+        print(response.content)
```

Comment:
is this needed?

Reply:
Not needed, but this is a test, so I feel it's fine to leave print statements like this (they will be filtered anyway **unless** the test fails).